api/routes.js

```diff
@@ -601,6 +601,7 @@ module.exports = (fastify, opt, next) => {
             host: req.headers['host'],
             event: eventNames[viewProgress],
             adId: adId,
+            time: new Date().toISOString()
```

Comment:
For clarity I would do `(new Date()).toISOString()`

Reply:
done